File: Data Structures & Algorithms/jump-game/submission-1.cs
public class Solution {
    public bool CanJump(int[] nums) {
        
        var max = 0;

        for(var i =0;i<nums.Length;i++){
            if(max < i) return false;

            max = Math.Max(max,i+nums[i]);
        }

        return true;
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let m = new Set()
        for(let n of nums){
            if(m.has(n)) return true
            m.add(n)
        }
        return false
    }
}
